internal_ws/ykshim/Cargo.toml

```diff
@@ -14,3 +14,6 @@ yksg = { path = "../yksg" }
 ykcompile = { path = "../ykcompile" }
 ykpack = { path = "../ykpack" }
 yktrace = { path = "../yktrace" }
+
+[features]
+testing = ["yksg/testing"]
```

Comment:
What's the significance of "yksg/testing" here?

Reply:
It means "when the testing flag for this crate is on, also build yksg with the testing flag."

Reply:
I see. I hadn't seen this before. The `/` makes it look like a filesystem path, but it's not.

Reply:
I'm not even sure if it's properly documented (from memory it's mentioned, but in a confusing manner), but I saw it somewhere, guessed what it did, and verified that it seems to match my guess.